README.md

```diff
@@ -1,16 +1,158 @@
+  - Generate a deployable container image from the converted application source code.
+  - Deploy the generated container image to the underlying runtime environment such as K8s, and automatically scale up and down according to business traffic, and scale to 0 when there is no traffic.
+  - Provide event management functions for trigger functions.
+  - Provide additional functions to manage function versions, traffic entry, etc.
+
+  ![](docs/images/OpenFunction-architecture.png)
+
+  ## Prerequisites
+
+  ---
+
+  The current version of OpenFunction requires that you have a Kubernetes cluster with version ``>=1.18.6``.
+
+  In addition, you need to deploy implementation components for the OpenFunction abstraction components ``Builder``, ``Serving``.
+
+  #### Builder
+
+  You can choose from the following options:
+
+  - OpenFunction Builder implementation based on Tekton and Cloud Native Buildpacks, you need to [install the Tekton project](https://tekton.dev/docs/getting-started/#installation) and then integrate it with the [Cloud Native Buildpacks project](https://buildpacks.io/docs/tools/tekton/).
+
+  #### Serving
+
+  You can choose from the following options:
+
+  - OpenFunction Serving implementation based on Knative, you need to [install the Knative project](https://knative.dev/docs/install/).
+
+  ## CustomResourceDefinitions
+
+  ---
+
+  The core functionality of OpenFunction is to serve users with the ability to develop, run, and manage business applications as execution units of code functions. OpenFunction is implemented based on the following [custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/):
+
+  - **```Function```**, abstract definition of a function.
+  - **```Builder```**, abstract definition of a function builder.
+  - **```Serving```**, abstract definition of a function workload.
+
+  ## QuickStart
+
+  ---
+
+  ### Install
+
+  You can quickly build an OpenFunction platform by executing the following command after selecting an appropriate version.
+
+  ```shell
+  kubectl apply -f config/bundle.yaml
+  ```
+
+  > Note: When using non-default namespaces, make sure that the ClusterRoleBinding in the namespace is adapted.
+
+  ### Sample: Run a function.
+
+  If you have already built implementations of OpenFunction Builder and OpenFunction Serving, follow steps below to run a simple case.
+
+  1. Creating a secret
+
+     In order to access your mirror repository, you need to create a secret. You can create this secret by editing the ``username`` and ``password`` fields in ``config/samples/registry-account.yaml``, and then apply it.
+
+     ```shell
+     kubectl apply -f config/samples/registry-account.yaml
+     ```
+
+  2. Creating functions
+
+     Here is a simple Function case for you, modify the ``spec.image`` field in ``config/samples/core_v1alpha1_function.yaml`` to your own repository address: 
```

Comment:
=>
For sample function below, modify the ``spec.image`` field in ``config/samples/core_v1alpha1_function.yaml`` to your own container registry address: